event-store/src/lib.rs

```diff
@@ -38,11 +38,10 @@ use chrono::prelude::*;
 pub use event::Event;
 pub use event_context::EventContext;
 use event_store_derive_internals::{EventData, Events};
-use futures::future::{ok as FutOk, Future};
+use futures::future::{ok as FutOk, result, Future};
```

Comment:
Can you call this `FutureResult` to keep with other futures stuff aliasing?

Reply:
Changes done